### PR TITLE
chore: SDA-1623: switch NodeJs dynamically during build

### PR DESCRIPTION
**Pull request by VishwasShashidhar**
## Description
Add support to switch nodejs dynamically during the build
[SDA-1623](https://perzoinc.atlassian.net/browse/SDA-1623)

## Solution Approach
Use NVM to switch between Node versions

## Related PRs
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "productName": "Symphony",
   "version": "6.0.0",
   "clientVersion": "2.0.1",
-  "buildNumber": "0",
+  "buildNumber":" 0",
```
Comment (KiranNiranjan) [inline package.json]: Is this change required?

Comment (VishwasShashidhar) [reply package.json]: Weird! I'll fix that.
